andela/ah-codeblooded-frontend
# #162199777 Handle 404 error (Page Not Found)

**What does this PR do?**

It handles the 404 or Page Not found error

**Description of Task to be completed?**

Whenever a user tries to access an undefined route, he or she is redirected to this specific page.

**How should this be manually tested?**

- Open this PR's review app
- Manoeuvre to any route of your choosing
- Edit route to a different one that does not exists
- You should see an Error Page

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

https://www.pivotaltracker.com/n/projects/2198535/stories/162199777

**Screenshots (if appropriate)**
Screenshot when someone tries to visit an undefined route

<img width="1440" alt="screen shot 2018-11-28 at 09 48 35" src="https://user-images.githubusercontent.com/26184534/49134387-6a0d5200-f2f4-11e8-83bb-1a29906037d3.png">

**Questions:**

N/A
